Lib/ssl.py

```diff
@@ -76,7 +76,12 @@
     PROTOCOL_TLSv1_2: 'TLSv1.2'
 }
 
-OP_ALL, OP_NO_SSLv2, OP_NO_SSLv3, OP_NO_TLSv1 = range(4)
```

Comment:
This really concerns me about the overall quality of the Jython codebase. You would think something like this would have been caught long ago. Or, you know, tested.